buffalo/cmd/filetests/apiapp.json

```diff
@@ -24,13 +24,7 @@
     "TemplatesBox",
     "Helpers"
   ]
-}, {
-  "path": "templates",
-  "absent": true
-}, {
-  "path": "assets",
-  "absent": true
-}, {
-  "path": "public",
+},{
+  "path": "public/assets/.dummy",
```

Comment:
This shouldn’t exist for API apps.

Reply:
Absent = true, so that’s what’s being asserted right ?

Reply:
Sorry, on a phone, didn’t see that. You’re correct. :)